error[E0609]: no field `wrote_div` on type `&mut Cpu`

I pulled down rust on a new system and tried to run `cargo build` just for fun. Below is the error I got:
```
220 |         rom_bank: 1,
    |         ----------- rustdoc does not generate documentation for expression fields
    |
    = help: use `/* */` for a plain comment

warning: unused doc comment
   --> src/memory.rs:223:7
    |
223 | /       /**
224 | |        * Gamepad default config
225 | |        */
    | |_________^
226 |         a: false,
    |         -------- rustdoc does not generate documentation for expression fields
    |
    = help: use `/* */` for a plain comment

error[E0609]: no field `wrote_div` on type `&mut Cpu`
   --> src/cpu.rs:367:10
    |
367 |     self.wrote_div = false;
    |          ^^^^^^^^^ unknown field
    |
    = note: available field is: `registers`

For more information about this error, try `rustc --explain E0609`.
warning: `gb_int` (lib) generated 3 warnings
error: could not compile `gb_int` (lib) due to 1 previous error; 3 warnings emitted
```

--- a/README.md
+++ b/README.md
@@ -4,7 +4,10 @@ An open source Gameboy emulator written in Rust that can use a command line inte
 
 ### Usage
 
-cargo run --release -- --rom PATH
+cargo run --release -- --rom PATH --mode terminal
+
+--mode options: 
+sdl|terminal|headless
 
 <p float="left">
   <img src="/screenshots/zelda1.png" width="49%" />

--- a/src/cpu.rs
+++ b/src/cpu.rs
@@ -234,12 +234,14 @@ impl Registers {
 #[derive(Serialize, Deserialize)]
 pub struct Cpu {
   pub registers: Registers,
+  pub wrote_div: bool,
 }
 
 impl Cpu {
   pub fn new() -> Self {
     Self {
       registers: Registers::default(),
+      wrote_div: false 
     }
   }
 

